index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap"
+        rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <header class="toolbar">
+        <a href="" class="logo">My Logo</a>
+        <nav class="navigation">
+            <img src="images/Icon-Menu.png" alt="menu_icon" id="icon-menu" class="hamburger">
+        </nav>
+    </header>
+    <main>
+        <section class="hero-section">
+
```

Comment:
[OPTIONAL] 
- [ ] You are missing the `section` closing tag, kindly add it to fulfill the best semantic structure of your code.